docs/apis/plugintypes/quizaccess/_examples/override_rule.php

```diff
@@ -0,0 +1,48 @@
+use mod_quiz\local\access_override_rule_base;
+use MoodleQuickForm;
+use context_module;
+use context;
+
+class override_rule extends access_override_rule_base {
+
+    /**
+     * All of the below access_override_rule_base functions to be implemented.
+     */
+
+    public static function add_form_fields(context_module $context, int $overrideid, object $quiz, MoodleQuickForm $mform): void {
```

Comment:
Example of exagerated line length limit:
```suggestion
    public static function add_form_fields(
        context_module $context,
        int $overrideid,
        object $quiz,
        MoodleQuickForm $mform,
    ): void {
```